tests/attacks/evasion/test_shadow_attack.py

```diff
@@ -61,15 +56,11 @@ def test_generate(fix_get_mnist_subset, get_image_classifier_list_for_attack):
 
         (x_train_mnist, y_train_mnist, x_test_mnist, y_test_mnist) = fix_get_mnist_subset
 
-        if attack.framework == "pytorch":
-            x_train_mnist = x_train_mnist.transpose((0, 3, 1, 2))
-
         x_train_mnist_adv = attack.generate(x=x_train_mnist[0:1], y=y_train_mnist[0:1])
 
         assert np.max(np.abs(x_train_mnist_adv - x_train_mnist[0:1])) == pytest.approx(0.34966960549354553, 0.06)
 
 
-@pytest.mark.only_with_platform("pytorch")
```

Comment:
Same thing as above. This test is using the framework independent fixture `get_image_classifier_list_for_attack` which is great. However based on the previous comment in `run_test.sh` this marker might need to be added if this test currently does not work for any framework other than pytorch `@pytest.mark.skipMlFramework("tensorflow", "keras", "scikitlearn")` Ideally, there should be no need for this marker since the test should work for all frameworks but in the event that it does not then this marker should be used withe the appropriate frameworks.

Reply:
I have added `@pytest.mark.skipMlFramework`.